Assets/Scripts/TileMapDemoScripts/CameraManager.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraManager
+{
+    private Camera mainCamera;
+    private const int CAMERA_BOUNDS = 1000;
+    private const int CAMERA_Z_POSITION = -10;
+    private const float SPEED = 20.0f;
+
+    public CameraManager(Camera mainCamera)
+    {
+        this.mainCamera = mainCamera;
+    }
+
+    public void UpdateCamera()
+    {
+        if (Input.GetKey(KeyCode.W)) {
```

Comment:
Would these all be if-elses? If the user presses "W" and "D" shouldn't both those blocks run to move the camera NE? And if the user presses "W" and "S" wouldn't it also be fine because the camera would just not move?

Also for readability, it may be good to blow these lines up a little bit.

Reply:
Good suggestion. The script has been updated to allow for multi-input camera controls.